packages/relay/src/services/UserService.ts

```diff
@@ -0,0 +1,137 @@
+import { SignupProof } from '@unirep/circuits'
+import { SnarkProof } from '@unirep/utils'
+import { UnirepSocialSynchronizer } from '../synchornizer'
+import TransactionManager from '../singletons/TransactionManager'
+import { APP_ADDRESS, CLIENT_URL } from '../config'
+import TwitterClient from '../singletons/TwitterClient'
+import crypto from 'crypto'
+import { UserRegisterStatus } from '../enums/userRegisterStatus'
+import { Contract } from 'ethers'
+import User from '../data/User'
+
+const STATE = 'state'
```

Comment:
Minor issue, why do we declare STATE here. Can we check state == 'state' direcly?